scripts/kata-install/osc-kata-install.sh

```diff
@@ -187,9 +187,66 @@ uninstall() {
 	wait_for_reboot_clear
 }
 
+get_cloud_provider() {
+	provider="$(kubectl get configmap/peer-pods-cm -n openshift-sandboxed-containers-operator -o jsonpath='{.data.CLOUD_PROVIDER}')"
+	
+	if [[ -z "${provider}" ]]; then
+		echo "Can't find the CLOUD_PROVIDER in the peer-pods-cm"
+		exit 1
+	fi
+	
+	echo "${provider}"
+}
+
+get_worker_node_version_ibmcloud() {
+	raw_version="$(kubectl get nodes/${NODE_NAME} -o jsonpath='{.metadata.labels.ibm-cloud\.kubernetes\.io/worker-version}')"
+
+	if [[ -z "${raw_version}" ]]; then
+		echo "Can't find the ibm-cloud.kubernetes.io/worker-version label"
```

Comment:
Errors should go to stderr. Since this script has quite some instances of that, I'd suggest to introduce an `error()` helper that writes to stderr and does `exit 1` (in a preparatory commit).